cmd/podman/common/netflags.go

```diff
@@ -159,9 +164,7 @@ func NetFlagsToNetOptions(cmd *cobra.Command) (*entities.NetOptions, error) {
 			return nil, err
 		}
 
-		return nil, errors.Errorf("network %s is not yet supported", network)
-		// TODO How do I convert a string network to a Specgen.Namespace?
-		//		opts.Network = specgen.Namespace{NSMode: network}
+		opts.Network = specgen.NewNamespace(network)
```

Comment:
This is really unlikely to work - `--net CNI1,CNI2,CNI3` will definitely not work. We need to parse this to check for the common ones (`host`, `bridge`, `slirp4netns`, `ns:$PATH` (which needs to compile into `NSMode: Path, Value: $PATH`) and then case anything else and populate the specgen's CNINetworks field with it